### DISCUSSION TITLE
2FA should be tested/monitored better!

### DISCUSSION THREAD

**Discussion by SomeTroglodyte** [community/community#76036] [Other Feature Feedback, Questions, & Ideas]
Link followed from a mail: https://github.com/yairm210/Unciv/pull/10443
Redirect "back" after 2FA: https://github.com/_view_fragments/voltron/pull_requests/show/yairm210/Unciv/10443/pull_request_layout

"Access to github.com was denied
You don't have authorization to view this page.
HTTP ERROR 403"

:rofl:  - no I didn't laugh at all.

Can you please explain to me what "voltron" means? You can't be referring to some 1980'es spam carrier?

Note: This wasn't a "regular" 2FA trigger, it said something about verifying whether I got it properly set up - hey look in your logs you clown, are three successful uses not enough?

**Comment by samus-aran** [comment]
Hi @SomeTroglodyte , 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- Reporting abuse or spam
- Reporting a user
- Reporting an issue or pull request
- Reporting a comment

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
